bitnami/nginx/values.yaml

```diff
@@ -146,15 +146,13 @@ resources:
 ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
 ##
 livenessProbe:
-  httpGet:
-    path: /
+  tcpSocket:
```

Comment:
Because when LDAP is enabled, the user may require requesters to authenticate. If the livenessProbe performs an HTTP GET request, it will end up in an `401 Unauthorized` and then, it will be considered as a failure.